Gemfile.lock

```diff
@@ -265,6 +273,7 @@ DEPENDENCIES
   debug
   devise
   dotenv-rails
+  faker!
```

Comment:
Are these changes automatically generated when you install the Faker gem? If this is manually added, what is the purpose?

Reply:
because i fake the email